lib/matplotlib/figure.py

```diff
@@ -949,8 +949,8 @@ def clf(self, keep_observers=False):
 
         self.stale = True
 
-    # synonym for `clf`."""
-    clear = clf
+    # synonym for `clear`.
+    clf = clear
```

Comment:
Coming back from the Axes discussion https://github.com/matplotlib/matplotlib/issues/22738#issuecomment-1084449395:

This needs to be repeated in all subclasses (was probably broken the other way round before). Do we know whether any 3rd party packages actually subclass Figure?

If so
```
def clf(self):
    self.clear()
```
may be the better choice.

OTOH it makes testing harder if those functions are not the same object (c.f. `test_clf_is_clear()`). A slightly fancy third way would be:
```
@property
def clf(self):
    return self.clear
```